apic_ml2/neutron/tests/unit/ml2/drivers/cisco/apic/test_cisco_apic_mechanism_driver.py

```diff
@@ -2010,6 +2010,29 @@ def setUp(self, service_plugins=None):
         # (but only for types not defined by the
         # mechanism driver class itself).
         self.driver.agent_type = ofcst.AGENT_TYPE_OPFLEX_OVS
+        self.driver.dvs_notifier = mock.MagicMock()
+        self.driver.dvs_notifier.bind_port_call = mock.Mock(
+            return_value=BOOKED_PORT_VALUE)
+
+    def _verify_dvs_notifier(self, notifier, port, host):
+            # can't use getattr() with mock, so use eval instead
+            try:
+                dvs_mock = eval('self.driver.dvs_notifier.' + notifier)
+            except Exception:
+                self.assertTrue(True,
+                                "The method " + notifier + " was not called")
+                return
+
+            dvs_mock.assert_called_once_with(
+                mock.ANY,
+                mock.ANY,
+                mock.ANY,
+                mock.ANY
+            )
+            a1, a2, a3, a4 = dvs_mock.call_args[0]
```

Comment:
The parameters passed are actually the port, and not the port ID. The port object that's passed to the call isn't easily available, and the main thing we care about is the port ID (i.e. are they doing the operation on the port they're supposed to).